sdk/spring/spring-cloud-azure-stream-binder-eventhubs/src/main/java/com/azure/spring/cloud/stream/binder/eventhubs/config/EventHubsBinderHealthIndicatorConfiguration.java

```diff
@@ -16,7 +16,7 @@
 @Configuration(proxyBeanMethods = false)
 @ConditionalOnClass(name = "org.springframework.boot.actuate.health.HealthIndicator")
 @ConditionalOnEnabledHealthIndicator("binders")
-class EventHubsBinderHealthIndicatorConfiguration {
+public class EventHubsBinderHealthIndicatorConfiguration {
```

Comment:
why changing this to public?

Reply:
It's used by `com.azure.spring.cloud.stream.binder.eventhubs.EventHubsBinderHints`, even though it is not supported yet.

Reply:
Should we still make it package-private, and make it public when it's supported?

Reply:
OK